packages/astro/src/core/build/static-build.ts

```diff
@@ -172,7 +176,16 @@ async function ssrBuild(
 					assetFileNames: `${settings.config.build.assets}/[name].[hash][extname]`,
 					...viteConfig.build?.rollupOptions?.output,
 					entryFileNames(chunkInfo) {
-						if (chunkInfo.facadeModuleId === resolvedPagesVirtualModuleId) {
+						if (chunkInfo.facadeModuleId?.startsWith(ASTRO_PAGE_RESOLVED_MODULE_ID)) {
+							return `${chunkInfo.facadeModuleId
```

Comment:
Can this renaming code be moved to its own function and some explanation for what it's doing?